qhub/template/stages/08-qhub-tf-extensions/modules/helm-extensions/variables.tf

```diff
@@ -27,6 +27,6 @@ variable "chart_version" {
 
 variable "overrides" {
   description = "Overrides for the helm chart values"
-  type        = list(any)
+  type        = any
   default     = []
```

Comment:
`default` should be `{}` otherwise this looks good to me!

Reply:
Updated it. Thanks